website/routes.py

```diff
@@ -1064,10 +1062,9 @@ def make_url_map(app):
     # Web
 
     process_rules(app, [
-        # '/' route loads home.mako if logged in, otherwise loads landing.mako
-        Rule('/', 'get', website_views.index, OsfWebRenderer('index.mako', trust=False)),
+        Rule('/', 'get', website_views.index, notemplate),
```

Comment:
Left the rules, but templates are now `notemplate`.  Views will either redirect to another view or use the ember app.